draft-ietf-masque-connect-udp-listen.md

```diff
@@ -110,14 +111,22 @@ target_port variables to the '*' character (ASCII character 0x2A).
 Before sending its UDP Proxying request to the proxy, the client allocates an
 even-numbered context ID, see {{Section 4 of CONNECT-UDP}}. The client then adds
 the "connect-udp-listen" header field to its UDP Proxying request, with its
-value set as the allocated context ID, see {{hdr}}.
+value set as the allocated context ID, see {{contextid}}. Likewise, the proxy
+can allocate and use odd numbered context IDs to send payloads to the client.
+
+The client and the proxy MUST exchange COMPRESSION_ASSIGN capsules in order to
```

Comment:
This part of the text is explaining the concept, so it doesn't need to be normative
```suggestion
The client and the proxy exchange COMPRESSION_ASSIGN capsules in order to
```